apps/permissions.go

```diff
@@ -20,6 +20,13 @@ const (
 	// receive permissions to any resources, need to be added explicitly.
 	PermissionActAsBot Permission = "act_as_bot"
 
+	// PermissionPostAllAsBot means that the Bot User will be assigned the role
+	// of `system_post_all`. This makes it so the bot can create posts in direct message
+	// and group message channels. Without this permission, the bot must be explicitly added to
```

Comment:
"This makes it so the bot can create posts in direct message and group message channels" - what other permissions the bot user gets by being assigned the `system_post_all` role?

Reply:
`PermissionReadWriteAllChannels` perhaps?